ocaml/xapi/xapi_pgpu.ml

```diff
@@ -29,6 +29,14 @@ let calculate_max_capacities ~__context ~pCI ~size ~supported_VGPU_types =
        vgpu_type, max_capacity)
     supported_VGPU_types
 
+let set_compatibility_metadata ~__context ~pgpu =
+  let pci = Db.PGPU.get_PCI ~__context ~self:pgpu in
+  if Db.PCI.get_vendor_id ~__context ~self:pci = Xapi_pci.id_of_int Xapi_vgpu_type.Nvidia.vendor_id then (
```

Comment:
The current one may be more efficient, if the compiler can replace `Xapi_pci.id_of_int Xapi_vgpu_type.Nvidia.vendor_id` with a constant (I'm not convinced it would).